site/maxlength.xml

```diff
@@ -56,8 +56,17 @@ limitations under the License.
         </p>
 
         <p>
-          Messages will be dropped or dead-lettered from the front of the
-          queue to make room for new messages once the limit is reached.
+          Queue overflow behaviour can be configured using the
+          <code>overflow</code> setting.
+          By default messages will be dropped or dead-lettered from the front
+          of the queue to make room for new messages, once the limit is reached.
```

Comment:
Is there a good way to make the default behavior more noticeable? The comments in rabbitmq/rabbitmq-server#995 imply that this behavior surprises RMQ users.